src/hb-ot-shape-complex-syllabic.hh

```diff
@@ -35,7 +35,8 @@ hb_syllabic_insert_dotted_circles (hb_font_t *font,
 				   hb_buffer_t *buffer,
 				   unsigned int broken_syllable_type,
 				   unsigned int dottedcircle_category,
-				   int repha_category = -1);
+				   int repha_category = -1,
+				   unsigned int dottedcircle_position = -1);
```

Comment:
Ah, since we are doing conditional assignment for `repha_category`, let's do the same for `dottedcircle_position`. Make it `int` and only use if not `-1`.

Can you also add a test please? Thanks!